src/get-started/java-binding.md

```diff
@@ -41,25 +30,42 @@ You can also install Exonum Java App manually:
   unzip /path/to/downloaded/exonum-java-0.6.0-release.zip
   ```
 
-
-### Linux
-
-- Install [Libsodium][libsodium] development library for your system.
-  You can use the following command for Ubuntu:
-
+- Install necessary runtime dependencies: [Libsodium][libsodium] and
+  RocksDB libraries (the latter is only needed on Mac OS):
+  
+  __Linux (Ubuntu)__
+  
   ```bash
   sudo apt-get update && sudo apt-get install libsodium-dev
   ```
-
-- Download and unpack the archive from the [Releases page][github-releases]
-  into some known location. To install the latest release to `~/bin`:
-
+  
+  __Mac OS__
+  
   ```bash
-  mkdir -p ~/bin
-  cd ~/bin
-  unzip /path/to/downloaded/exonum-java-0.6.0-release.zip
+  brew install libsodium rocksdb
   ```
 
+- Follow steps in [After Install](#after-install) section below
```

Comment:
Follow the steps in the [After Install](#after-install) section below.

Reply:
JFYI, it might have been purged in #331